validator/gulpjs/test/validate.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * @license
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the license.
+ */
+
+'use strict';
+
+const assert = require('assert');
+const es = require('event-stream');
+const gutil = require('gulp-util');
+const File = require('vinyl');
+const fs = require('fs');
+const amphtmlValidator = require('../');
```

Comment:
Maybe I'm looking at the wrong version of the code, it still says amphtmlValidator here? No worries if it's intentional.